examples/07-EMIT/EMIT_HFSS_Example.py

```diff
@@ -0,0 +1,139 @@
+# 
+# The following code uses AEDT 2023 R1.
+
+non_graphical = False
+NewThread = True
+desktop_version = "2023.2"
+scratch_path = pyaedt.generate_unique_folder_name()
+
+###############################################################################
+# Launch AEDT with EMIT
+# ~~~~~~~~~~~~~~~~~~~~~
+# Launch AEDT with EMIT. The ``Desktop`` class initializes AEDT and starts it
+# on the specified version and in the specified graphical mode.
+
+d = pyaedt.launch_desktop(desktop_version, non_graphical, NewThread)
+
+temp_folder = os.path.join(scratch_path, ("EmitHFSSExample"))
+if not os.path.exists(temp_folder):
+    os.mkdir(temp_folder)
+
+example_name = "Cell Phone RFI Desense"
+example_aedt = example_name + ".aedt"
+example_results = example_name + ".aedtresults"
+example_lock = example_aedt + ".lock"
+example_pdf_file = example_name + " Example.pdf"
+
+example_dir = os.path.join(d.install_path, "Examples\\EMIT")
+example_project = os.path.join(example_dir, example_aedt)
+example_results_folder = os.path.join(example_dir, example_results)
+example_pdf = os.path.join(example_dir, example_pdf_file)
+
+########################################################################################################
+# If the ``Cell Phone RFT Defense`` example is not
+# in the installation directory, exit from this example.
+
+if not os.path.exists(example_project):
+    msg = """
+        Cell phone RFT Desense example file is not in the
+         Examples/EMIT directory under the EDT installation. You cannot run this example.
+        """
+    print(msg)
+    d.release_desktop(True, True)
+    exit()
+
+my_project = os.path.join(temp_folder, example_aedt)
+my_results_folder = os.path.join(temp_folder, example_results)
+my_project_lock = os.path.join(temp_folder, example_lock)
+my_project_pdf = os.path.join(temp_folder, example_pdf_file)
+
+if os.path.exists(my_project):
+    os.remove(my_project)
+
+if os.path.exists(my_project_lock):
+    os.remove(my_project_lock)
+
+with Scratch(scratch_path) as local_scratch:
+    local_scratch.copyfile(example_project, my_project)
+    local_scratch.copyfolder(example_results_folder, my_results_folder)
+    if os.path.exists(example_pdf):
+        local_scratch.copyfile(example_pdf, my_project_pdf)
+
+aedtapp = pyaedt.Emit(my_project)
+
+###############################################################################
+# Create and connect EMIT components
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Create two radios with antennas connected to each one.
+
+rad1, ant1 = aedtapp.modeler.components.create_radio_antenna("Bluetooth Low Energy (LE)")
+rad2, ant2 = aedtapp.modeler.components.create_radio_antenna("Bluetooth Low Energy (LE)")
+
+###############################################################################
+# Define coupling among RF systems
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Define coupling among the RF systems.
+
+for link in aedtapp.couplings.linkable_design_names:
+    aedtapp.couplings.add_link(link)
```

Comment:
@jsalant22 I know that we had this example before, but reviewing it now, I'm wondering if the coupling link is used. Do we associate the linked coupling with antennas anywhere?